pokerole_irl/worlds_app/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import SettingListView, SettingCreateView, SettingView, CharacterView, CharacterCreateView
+
+urlpatterns = [
+    path('setting_list/', SettingListView.as_view(), name='setting-list'),
+    path('setting_create/', SettingCreateView.as_view(), name='setting-create'),
+    path('setting/<int:pk>', SettingView.as_view(), name='setting'),
+    path('character_create/', CharacterCreateView.as_view(), name='character-create'),
+    path('character/<int:pk>', CharacterView.as_view(), name='character')
```

Comment:
Remember the trailing "/" in the URL

Reply:
This should also be in a setting "<int:setting_pk>/character/<int:pk>/"

Reply:
Need some help here. I don't know how to get the value of the setting from the previous HTML page

Reply:
Can you elaborate a bit? Are you having trouble passing along the pk of the object in the link in the HTML?